samples/NativeLibrary/README.md

```diff
@@ -28,17 +28,97 @@ The above command will drop a static library (Windows `.lib`, OSX/Linux `.a`) in
 
 The above command will drop a shared library (Windows `.dll`, OSX `.dylib`, Linux `.so`) in `./bin/[configuration]/netstandard2.0/[RID]/publish/` folder and will have the same name as the folder in which your source file is present.
 
+### Loading shared libraries from C and importing methods
+For reference, you can read the two C files located in the c_source folder.
+The first thing you'll have to do in order to have a proper "loader" that loads your shared library is to add these directives
+
+```c
+    #ifdef _WIN32
+    
+    #include "windows.h"
+    
+    #else
+    
+    #include "dlfcn.h"
+    
+    #endif
+    
+    #include "stdio.h"
+    
+    #include "string.h"
+    
+    #ifdef _WIN32
```

Comment:
This can be made more compact by just putting everything under a single ifdef. 
```
#ifdef _WIN32
#include "windows.h"
#define __symLoad GetProcAddress GetProcAddress
#define __libClose FreeLibrary
#else
#include "dlfcn.h"
#define __symLoad dlsym
#define __libClose dlclose
#endif
```